### PR TITLE
Skip profile.d in a one-off dyno

### PR DESCRIPTION
**Pull request by jkutner**

### CODE REVIEW

**Comment by jkutner** [comment]
Fixes #12

File: .profile.d/heroku-metrics-daemon.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # don't do anything if we don't have a metrics url.
-if [[ -z "$HEROKU_METRICS_URL" ]]; then
+if [[ -z "$HEROKU_METRICS_URL" ]] || [[ "${DYNO}" = *run.* ]]; then
```
Comment (apg) [inline .profile.d/heroku-metrics-daemon.sh]: Why `*run.*`? It's always run.*, no?

Comment (jkutner) [reply .profile.d/heroku-metrics-daemon.sh]: yup

File: .profile.d/heroku-metrics-daemon.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # don't do anything if we don't have a metrics url.
-if [[ -z "$HEROKU_METRICS_URL" ]] || [[ "${DYNO}" = *run.* ]]; then
+if [[ -z "$HEROKU_METRICS_URL" ]] || [[ "${DYNO}" = run.* ]]; then
```
Comment (freeformz) [inline .profile.d/heroku-metrics-daemon.sh]: should it be `run\.*` ? I'm not sure if the . is interpreted as a `.` or any single character.

**Pull request merged**